django_lti_tool_provider/models.py

```diff
@@ -57,3 +61,36 @@ def send_lti_grade(self, grade):
         ))
 
         return outcome
+
+    @classmethod
+    def get_or_create_by_parameters(cls, user, authentication_manager, lti_params):
+        """
+        Filters out OAuth parameters
+        """
+        custom_key = authentication_manager.vary_by_key(lti_params)
+
+        # implicitly tested by test_views
+        if custom_key is None:
+            custom_key = ''
+
+        lti_user_data, created = LtiUserData.objects.get_or_create(user=user, custom_key=custom_key)
+
+        if lti_user_data.edx_lti_parameters.get('user_id', lti_params['user_id']) != lti_params['user_id']:
+            # TODO: not covered by test
```

Comment:
@Kelketek nit: not sure, but it might actually be implicitly covered by `test_force_login_change`: `LTIView._right_user`. Anyway, explicit test would still be nice to have, so let it be :)